source/_integrations/husqvarna_automower.markdown

```diff
@@ -113,7 +113,14 @@ The integration will create a lawn mower entity to control the mower. This entit
 
 ### Number (if available)
 
-The integration will create a number entity for changing the cutting height of the mower. This entity is disabled by default. You have to enable it manually because it can't be detected with the API if the mower has the capability to change the cutting height remotely. Before enabling this function, consult the handbook of the mower. It's possible that you can use this entity only as sensor but cannot actively change the cutting height.
+#### Cutting height
+
+The integration will create a number entity for changing the cutting height of the mower. This entity is disabled by the default. You have to enable it manually, because it can't be detected with the API if the mower has the capabilities to change the cutting height remotely. Consult the handbook of the mower before you enable this function. Also it's possible, that you can use this entity only as sensor and not actively change the cutting height.
+Possible values are 1 to 9.
+
+#### Cutting height for work areas
+
+The integration will create a number entity for changing the cutting height for each work area of the mower if your mower supports work areas. Possible values are 0% to 100%.
```

Comment:
```suggestion
The integration will create a number entity for changing the cutting height for each work area of the mower if your mower supports work areas. Possible values are 0% to 100%.
```
What does the percentage indicate? If I put 40, will it mow only 40% of the area? Or cut the grass down to 40% of it's current length? Or cut 40% of the current length and leave 60% standing?

Reply:
I've detailed the description.

Reply:
Thank you for specifying.